polling_stations/templates/base.html

```diff
@@ -27,21 +27,27 @@
             <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
             <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 
+            <link rel="shortcut icon" href="{%  static 'images/favicon.ico' %}">
```

Comment:
I now don't have any favicon on Firefox on Ubuntu, I don't know what the issue is, but I think it's better to break it for IE11 users than for firefox users.